aten/src/ATen/native/SpectralOpsUtils.h

```diff
@@ -6,10 +6,30 @@
 
 namespace at { namespace native {
 
-// Since real-to-complex satisfy the conjugate symmetry, i.e.,
-// X[m, \omega] = X[m, N - \omega]*. We return only the first floor(N / 2) + 1
-// values by default (onesided=True). This is also the assumption in libraries
-// including cuFFT and MKL.
+// [ NOTE ] Fourier Transform Congjugate Symmetry
```

Comment:
Nit; the prevailing style is `Note [Blah blah blah]`. But if you're consistent at use-site and decl-site I don't mind.

Reply:
Also, spelling.